### PR TITLE
Add description for upgrade_hw_version property

### PR DESCRIPTION
**Pull request by EleanorRigby**

### CODE REVIEW

**Comment by EleanorRigby** [comment]
@yeshwantbabar

File: content/vsphere-cpi.md [hunk 1]
```diff
@@ -189,6 +190,7 @@ Schema:
     * **password** [String, required]: The login password for the NSX server.
     * **ca_cert** [String, optional]: A CA certificate that can authenticate the NSX server certificate. **Required** if the NSX Manager has a self-signed SSL certificate. Must be in PEM format.
 * **enable\_auto\_anti\_affinity\_drs\_rules** [Boolean, optional]: Creates DRS rule to place VMs on separate hosts. DRS Automation Level must be set to "Fully Automated"; does not work when DRS is set to "Partially Automated" or "Manual". May cause VMs to fail to power on if there are more VMs than hosts after initial deployment. Default: `false`. Available in v33+.
+* **upgrade\_hw\_version** [Boolean, optional]: Upgrades the virtual hardware version of a virtual machine to the latest supported version on the ESXi host. This property can also be specified in VM Types which will override the global configuration. By default this property is false.
```
Comment (nehagjain15) [inline content/vsphere-cpi.md]: This should be just:
 Upgrades the hardware version of a virtual machine to the latest supported version on the ESXi host. Default: `false`

Comment (EleanorRigby) [reply content/vsphere-cpi.md]: What about override information?

File: content/vsphere-cpi.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ Schema for `cloud_properties` section:
 * **disk** [Integer, required]: Ephemeral disk size in megabytes. Example: `10240`.
 * **cpu\_hot\_add\_enabled** [Boolean, optional]: Allows operator to add additional CPU resources while the VM is on. Default: `false`. Available in v21+.
 * **memory\_hot\_add\_enabled** [Boolean, optional]: Allows operator to add additional memory resources while the VM is on. Default: `false`. Available in v21+.
+* **upgrade\_hw\_version** [Boolean, optional]: Upgrades the virtual hardware version of a virtual machine to the latest supported version on the ESXi host. This property can also be specified in global configuration which will be overridden by VM Type property. By default this property is false.
```
Comment (nehagjain15) [inline content/vsphere-cpi.md]: Upgrades virtual machines to latest virtual hardware version supported on the ESXi host. Overrides the global upgrade\_hw\_version.  Available in v49+ ?

File: content/vsphere-cpi.md (hunk 1 above)
Comment (nehagjain15) [reply content/vsphere-cpi.md]: Override information is provided in cloud config section. The usual syntax is to describe what the property is here and if its being overridden by cloud config, you specify there which property it is overriding

**Pull request merged**